src/main/java/org/owasp/wrongsecrets/challenges/Challenge.java

```diff
@@ -65,7 +59,12 @@ public abstract class Challenge {
      * @return true if answer was correct.
      */
     public boolean solved(String answer) {
-        var correctAnswer = answerCorrect(answer);
+        boolean correctAnswer = false;
```

Comment:
can you please revert all changes to the Challenge.java class? let's change your implementation a liiiitle bit